proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/connector/hbase/HBaseBackgroundExecutorManager.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.proxy.backend.connector.hbase;
+
+import lombok.Getter;
+import org.apache.shardingsphere.infra.executor.kernel.thread.ExecutorThreadFactoryBuilder;
+import java.io.Closeable;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * background executor.
+ */
+@Getter
+public class HBaseBackgroundExecutorManager implements Closeable {
+    
+    private final ScheduledExecutorService executorService;
+    
+    public HBaseBackgroundExecutorManager() {
+        executorService = getExecutorService();
+    }
+    
+    private ScheduledExecutorService getExecutorService() {
+        ThreadFactory threadFactory = ExecutorThreadFactoryBuilder.build("background");
+        return Executors.newScheduledThreadPool(1, threadFactory);
+    }
+    
+    /**
+     * submit background task.
```

Comment:
The first letter should be upper case, and please keep a blank line between JavaDoc and @param